src/reportSources/legacyFflogs/eventAdapter/translate.ts

```diff
@@ -99,7 +99,7 @@ export class TranslateAdapterStep extends AdapterStep {
 			return [this.adaptTargetableEvent(baseEvent)]
 
 		case 'combatantinfo':
-			return this.adaptCombatantInfoEvent(baseEvent)
+			return this.adaptCombatantInfoEvent(baseEvent).concat(this.adaptCombatantInfoEventAuras(baseEvent))
```

Comment:
this should all be handled in the main `adaptCombatantInfoEvent` method

if the logic is too clunky to be in one method, feel free to branch out from within there - but it shouldn't be doing non-trivial logic in the primary adapt switch